### ISSUE TITLE
dataset : nsmc - ResourceExhaustedError ?

### ISSUE THREAD

**Issue by jiwoongim** [carpedm20/lstm-char-cnn-tensorflow#10]
I have a question about running on nsmc dataset
python main.py --dataset nsmc doesn't seem to work on both gtx1080 (8 GB)  and k40 (12 GB).
I get such an error
" ... tensorflow.python.framework.errors.ResourceExhaustedError: OOM when allocating tensor with shape[100,450537]  ..."
Is this "python main.py --dataset nsmc" supposed to run on a specific gpu? are there certain requirements?

**Comment by carpedm20** [comment]
No, I tested this code on gtx980ti so 1080 should fit well. But I've never tried `nsmc` before because ptb still couldn't converge well as I discussed in #3. Was it okay to run `ptb` instead of `nsmc`?

**Comment by jiwoongim** [comment]
Python main.py doesn't crush and runs on GTX1080. However it fails to train at all. When I trained on k40, python main.py works.  I get "{'perplexity': 88.218780156903421, 'epoch': 24, 'valid_perplexity': 150.16004922594567, 'learning_rate': 0.0001220703125}

> Test: loss: 4.916528, perplexity: 136.527725
>  [*] Test loss: 4.972944, perplexity: 144.451502"
